ktor-server/ktor-server-host-common/api/ktor-server-host-common.api

```diff
@@ -56,6 +56,10 @@ public final class io/ktor/server/engine/ApplicationEngineEnvironmentBuilder {
 	public final fun setWatchPaths (Ljava/util/List;)V
 }
 
+public final class io/ktor/server/engine/ApplicationEngineEnvironmentKt {
+	public static final fun applicationEngineEnvironment (Lkotlin/jvm/functions/Function1;)Lio/ktor/server/engine/ApplicationEngineEnvironment;
+}
+
```

Comment:
No idea why this has changed...

Reply:
It seems it's still failing in TC, something funky is going on with this module.

Reply:
Good news: that gives me what's expected on TC 🥳 
Bad news: there's still a bunch of mysterious changes in that API that have nothing to do with my changes 😕 
Good news: the API dump is the same in main 🎉 
Bad news: something was merged with a big API update that wasn't picked up in the CI 😱

Reply:
that's fine, this module was dropped